e2e-tests/src/meroctl.rs

```diff
@@ -249,16 +250,43 @@ impl Meroctl {
 
         command_line.push('\'');
 
-        self.output_writer.write_str(&command_line);
+        output_writer.write_str(&command_line);
 
         async move {
             let output = command
                 .stdout(Stdio::piped())
+                .stderr(Stdio::piped())
                 .spawn()?
                 .wait_with_output()
                 .await?;
 
-            Ok(serde_json::from_slice(&output.stdout)?)
```

Comment:
can you revert this, I just expected you to print the string representation of `output.stdout`

since it was failing on a deserialization error

as for `stderr`, we don't care about capturing it, by default it'll inherit from the parent which is acceptable